apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/dashboard/HomeDashboard.jsx

```diff
@@ -399,36 +346,39 @@ function HomeDashboard() {
     const summaryInfo = [
         {
             title: 'Total APIs',
-            data: transform.formatNumberWithCommas(totalAPIs),
+            data: transform.formatNumberWithCommas(totalAPIs[totalAPIs.length-1]),
```

Comment:
this and below, wherever we're using arr[arr.length-1], handle case where arr.length is 0 and arr is undefined